src/libs/ErrorUtils.js

```diff
@@ -54,8 +54,30 @@ function getLatestErrorMessage(onyxData) {
         .value();
 }
 
+/**
+     *
+     * @param {Object} errors - An object containing current errors in the form
+     * @param {String} key - inputID of the field
+     * @param {String} message - Message to assign to the inputID errors
+     * @returns {Object} - An object containing the errors for each inputID
+     */
```

Comment:
The indentation is a bit off based on what we normally do. Also it would be nice to have a brief description of what this method does.